docs/managing-findings.md

```diff
@@ -50,57 +51,24 @@ severity:warning rule:finding-test: Finding test 2
 1:print(1 == 1)
 ```
 
-For more information on writing rules, see [Rule syntax](../writing-rules/rule-syntax/).
+For more information on writing rules, see [Rule syntax](/writing-rules/rule-syntax/).
 
 ## Semgrep CI
 
-[Semgrep CI](../semgrep-ci/overview/), designed to continuously scan commits and builds, improves on Semgrep findings to track the lifetime of an individual finding. A Semgrep CI finding is defined by a 4-tuple:
-
-```
-(rule ID, file path, syntactic context, index)
-```
-
-These pieces of state correspond to:
-
-1. `rule ID`: the rule's ID within the Semgrep ecosystem.
-1. `file path`: the filesystem path where the finding occurred.
-1. `syntactic context`: the lines of code corresponding to the finding.
-1. `index`: an index into identical findings within a file. This is used to disambiguate findings.
-
-:::info
-`syntactic context` is normalized by removing indentation, [`nosemgrep`](../ignoring-files-folders-code/#ignoring-code-through-nosemgrep) comments, and whitespace.
-:::
-
-These are hashed and returned as the syntactic identifier: `syntactic_id`. This is how Semgrep CI uniquely identifies findings and tracks them across state transitions. Semgrep CI does not store or transmit code contents. The `syntactic context` is hashed using a one-way hashing function making it impossible to recover the original contents.
+[Semgrep CI](/semgrep-ci/overview/), designed to continuously scan commits and builds, improves on Semgrep findings to track the lifetime of an individual finding.
```

Comment:
@chmccreery I removed all of the information above (removed lines 57-74) because I am not sure that this is really important for our users. Do you think that this was also exposing our internal processes? If this is somehow useful to users, let me know if we should put it back.